oshi-core/src/main/java/oshi/util/EdidUtil.java

```diff
@@ -201,6 +201,19 @@ public static String getDescriptorRangeLimits(byte[] desc) {
     public static String getDescriptorText(byte[] desc) {
         return new String(Arrays.copyOfRange(desc, 4, 18), StandardCharsets.US_ASCII).trim();
     }
+    /**
+     * Get the preferred resolution for the monitor (Eg: 1920x1080)
+     *
+     * @param edid The edid Byte array
+     * @return Plain text preferred resolution
+     */
+
+    public static String getPreferredResolution(byte[] edid) {
+        int dtd = 54;
+        int horizontalRes = (edid[dtd + 4] & 240) << 4 | edid[dtd + 2] & 255;
+        int verticalRes = (edid[dtd + 7] & 240) << 4 | edid[dtd + 5] & 255;
```

Comment:
Can we change the bitmasking to `& 0xFF` (or `0xff`) for consistency with other usages in this file?

Reply:
Yes, we need tests. Hopefully you can grab a real EDID to get the expected output.